.sbtrc

```diff
@@ -0,0 +1,2 @@
+alias test=; testOnly -- -l IntegrationTest
+alias testDB=; testOnly -- -n IntegrationTest
```

Comment:
This change in wrong direction compared to previous commits, IMHO.
I would like:
* `test` should skip anything not tagged _IntegrationTest_ (I guess `-l` means "ignore"?)
* `testIT` should do all tests tagged _IntegrationTest_ (I guess `-n` means "ignore"?)
* `testDB` should run all tests in database module (in this regard my prvious ocmment was missunderstood)

The fact that all tests marked  _IntegrationTest_ are currently testing database is just a coincidence, does not justify the alias name.

Reply:
`test` - Yes, `-l` does not mean ignore. The ignore is an in-build Tag that controls which tests do not run, but it is permanent from the test developer's perspective.

In the atum-service project, we use two test frameworks (ScalaTest and Zio), which we cannot control using the same logic/command solution. Using two commands (`testIT`, `testDB`) here helps separate these parallel solutions.

We can follow these differences and use DBTestTag for the database module.